### ISSUE TITLE
Using asp.net core 2.0.0, does not seem to correctly instantiate the MailChimpManager

### ISSUE THREAD

**Issue by SneezeWeeze** [brandonseydel/MailChimp.Net#247]
```
                    MailChimpManager humblebum = new MailChimpManager(
                       apiKey: "you api key here");
                    var listId = "list name here";
                    var member = new Member
                    {
                        EmailAddress = $"githubTestAccount@test.com",
                        StatusIfNew = Status.Subscribed
                    };
                    member.MergeFields.Add("FNAME", "HOLY");
                    member.MergeFields.Add("LNAME", "COW");
                 await humblebum.Members.AddOrUpdateAsync(listId, member);
```

I added a break after the instantiation of the MailChimpManager to see the configuration properties for the API and its showing these errors:

`'((MailChimp.Net.MailChimpConfiguration)((MailChimp.Net.Core.BaseLogic)humblebum.Api)._mailChimpConfiguration).ApiKey' threw an exception of type 'System.IO.FileNotFoundException'`

`'((MailChimp.Net.MailChimpConfiguration)((MailChimp.Net.Core.BaseLogic)humblebum.Api)._mailChimpConfiguration).AuthHeader' threw an exception of type 'System.IO.FileNotFoundException'`

`'((MailChimp.Net.MailChimpConfiguration)((MailChimp.Net.Core.BaseLogic)humblebum.Api)._mailChimpConfiguration).DataCenter' threw an exception of type 'System.IO.FileNotFoundException'
`

Is anyone else experiencing issues? If so, know how to solve them? It could have nothing to do with asp.net core 2.0.0

**Comment by Ellfish** [comment]
I'm also getting this exception with .NET Core 1.1, full exception message being: 

System.IO.FileNotFoundException: Could not load file or assembly 'System.Net.Http.Formatting, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.

Seems to be because System.Net.Http.Formatting isn't in .NET Core: https://github.com/aspnet/Mvc/issues/5562

**Comment by ctolkien** [comment]
This library is currently not NetStandard compatible. It won't work on ASPNET Core. Recent commits have gone in which look to fix this however:

See: https://github.com/brandonseydel/MailChimp.Net/pull/203

**Comment by brandonseydel** [comment]
This has been released as version 3.0.0 on nuget

**Comment by Ellfish** [comment]
Thanks @brandonseydel (and other contributors), much appreciated!
